staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -4772,6 +4772,7 @@ func describePriorityClass(pc *schedulingv1.PriorityClass, events *corev1.EventL
 		w.Write(LEVEL_0, "Name:\t%s\n", pc.Name)
 		w.Write(LEVEL_0, "Value:\t%v\n", pc.Value)
 		w.Write(LEVEL_0, "GlobalDefault:\t%v\n", pc.GlobalDefault)
+		w.Write(LEVEL_0, "PreemptionPolicy:\t%s\n", *pc.PreemptionPolicy)
```

Comment:
Since `PreemptionPolicy` is optional, you're going to want to check for nil and display `<unset>` in that case.  See other examples in `describe.go`, like this one:

https://github.com/kubernetes/kubernetes/blob/e7e2bfde16381dc9ae543123ebbe1da8c12ee8e4/staging/src/k8s.io/kubectl/pkg/describe/describe.go#L4808-L4812

Reply:
Thanks for your review. @brianpursley  
I am not sure if I have already set the default value for PreemptionPolicy if it's nil. Do I still need to check for nil when describe.

https://github.com/kubernetes/kubernetes/blob/ff4f560238290f5e918aa65c5c381690817aea52/pkg/apis/scheduling/v1/defaults.go#L31-L36

Reply:
Ah yeah, this should be OK